command/server/config.go

```diff
@@ -905,3 +905,120 @@ func parseTelemetry(result *Config, list *ast.ObjectList) error {
 
 	return nil
 }
+
+// Sanitized returns a copy of the config with all values that are considered
+// sensitive stripped. It also strips all raw values that are mainly
+// used for parsing.
+//
+// Specifically, the fields that this method strips are:
+// - Storage.Config
```

Comment:
It feels like stripping out Storage.Config is extreme.  I understand that it's impractical to be more fine-grained in general, but our enterprise customers are mostly going to be using Consul and Raft.  Can we special-case those and only strip out the truly sensitive config, e.g. consul token?

Reply:
If we were going to show certain values from Config, I think the safer approach would be to have an allow-list rather than a set of values to strip, as they could be easily overlooked/forgotten since it's better to forget adding a non-sensitive value than leaking a sensitive one. I started down this road initially, but stopped after realizing the numerous types of storage backends that we support and the number of fields that we'd need to include for each of those.

I think the cleaner approach would be to have a separate endpoint that displayed the complete configuration file params, including the Config map, (and that could be included in the debug bundle future release), but didn't want to add this prematurely.